README.md

```diff
@@ -63,6 +63,7 @@ Within the file you can configure the following properties:
 |---             |---                                      |---    |
 |`github.api.url`|URL of the Github API to operate against.|`https://api.github.com`|
 |`github.principal.cache.ttl`|[Java Duration](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#parse-java.lang.CharSequence-) for how long a given Access will be cached for. This is a tradeoff of how quickly access can be revoked and how quickly a Github user's rate limit will be reached for the Github User API. _Note:_ Github Enterprise does not have a rate limit!|`PT1M` (1 Minute)|
+|`github.org`|The Organization the user should be a member of. If this is not set anyone with a Github account is allowed tot login.|----|
```

Comment:
Pls remove this `Note that anybody is allowed to login (authenticate) with a valid Github Token from your Github instance, but he/she won't have any priviledges assigned with their teams (authorization).` from the docs :-) Thanks in advance!

Reply:
+1